### ISSUE TITLE
Transforming code with ignored destructure array values, results in fatal error

### ISSUE THREAD

**Issue by somombo** [lebab/lebab#236]
[As shown here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Ignoring_some_returned_values), the following is perfectly valid ES6/7 code ...

#### Given:

```typescript  
// ignore_array_values.js

var [,x] = ["hello","world"];

console.log(x);  // expect `world`
```

#### However, at the CLI, if you run:

```lebab ignore_array_values.js --transform let```

#### The result is, error:

```
  if (node.type === 'Identifier') {
          ^

TypeError: Cannot read property 'type' of null
    at extractVariables (%APPDATA%\npm\node_modules\lebab\lib\utils\destructuring.js:26:11)
    at arrayMap (%APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:660:23)
    at Function.map (%APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:9571:14)
    at interceptor (%APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:16970:35)
    at thru (%APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:8812:14)
    at %APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:4379:28
    at arrayReduce (%APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:704:21)
    at baseWrapperValue (%APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:4378:14)
    at LazyWrapper.lazyValue [as value] (%APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:1880:16)
    at baseWrapperValue (%APPDATA%\npm\node_modules\lebab\node_modules\lodash\lodash.js:4376:25)
```  

#### Note:

- When the same command is run using **_any other_** `transforms` on the **same** file, it does not result in any error.
- I tested this only on windows 10 with node v7.6.0, npm v4.3.0 and lebab v2.7.2

**Comment by nene** [comment]
Thanks for reporting. Reproduces just fine. Will investigate what might be causing this.

**Comment by nene** [comment]
Should be fixed now. Fix released in Lebab 2.7.7.

**Comment by somombo** [comment]
Thank you!
